packages/zendesk-adapter/src/change_validators/custom_statuses_enabled.ts

```diff
@@ -60,6 +60,75 @@ const areCustomStatusesEnabled = async (
   return customStatusesEnabled.enabled
 }
 
+const createErrorsForCustomStatusTypes = (
+  changes: readonly Change<ChangeDataType>[]
+): ChangeError[] =>
+  changes
+    .filter(
+      change =>
+        getChangeData(change).elemID.typeName === CUSTOM_STATUS_TYPE_NAME
+    )
```

Comment:
1. why not
```suggestion
    .filter(change => getChangeData(change).elemID.typeName === CUSTOM_STATUS_TYPE_NAME)
```
2. this seems like existing code, but I think we might want different handling for removals? (e.g. just silently ignore the deletions?) @edenhassid wdyt?

Reply:
1. Done 👍 (I ran prettier with default config on this block which limits to 80 characters, I'll fix that in my setup)

Reply:
I think deletion should also be included, I don't think its possible to delete a custom status if the feature is disabled